znai-server/src/main/java/com/twosigma/znai/server/DocumentationServer.java

```diff
@@ -198,7 +229,23 @@ private static boolean isNotDocPageRequest(RoutingContext ctx) {
         return !extension.equals("html");
     }
 
-    private void unzip(String docId, Path path) {
+    private void deleteCachedDocument(String docId, String actor) {
+        Path docPath = deployRoot.resolve(docId);
+
+        ConsoleOutputs.out(Color.BLUE, "deleting docs: ", Color.PURPLE, docId, Color.BLACK, " at ",
+                Color.PURPLE, docPath);
+        try {
+            File docDirectory = docPath.toFile();
+            if (docDirectory.exists()) {
+                org.apache.commons.io.FileUtils.deleteDirectory(docDirectory);
```

Comment:
probably should remove fully qualified name

Reply:
I used a fully qualified name due to com.twosigma.znai.utils.FileUtils.